docs/source/API/core/KokkosConcepts.rst

```diff
@@ -120,10 +119,11 @@ and ``OpenMPTarget``, the current state of the Kokkos |ExecutionSpaceTwo|_ conce
             typename Ex::device_type;
         } &&
         // Required methods:
-        requires(Ex ex, std::ostream& ostr, bool detail) {
-            { ex.in_parallel() } -> bool;
+        requires(Ex ex, std::string str, std::ostream& ostr, bool detail) {
+            { ex.fence(str) };
             { ex.fence() };
             { ex.name() } -> const char*;
+            { ex.concurrency() } -> int;
```

Comment:
Remind me, did we promise anything about the return type?

Reply:
That's how its implemented and how we describe it in all documentation so far. I don't know if there would be any motivation to change it to `size_type` or something the like.

Reply:
I meant more returns an integral type or something that is convertible to int.
Anyway that's fine by me if all backend do indeed return an int